src/main/java/lotto/domain/Rank.java

```diff
@@ -0,0 +1,44 @@
+package lotto.domain;
+
+import java.util.Arrays;
+
+public enum Rank {
+
+    FIRST(6, 2000000000),
```

Comment:
enum 을 잘 활용해주셨네요, 숫자 가독성을 위해 아래처럼 표현해보시는걸 추천드립니다 :)

```suggestion
    FIRST(6, 2_000_000_000),
```

Reply:
변경완료했습니다!